### PR TITLE
Create TempURL sample code for Gophercloud

### PR DESCRIPTION
**Pull request by jrperritt**

### CODE REVIEW

File: src/docs/cloud-files/getting-started/samples/get_object_temp_url.rst [hunk 1]
```diff
@@ -4,7 +4,11 @@
 
 .. code-block:: go
 
-  // Not currently supported by this SDK
+  createTempURLOpts := osObjects.CreateTempURLOpts{
+    Method: osObjects.GET,
+    TTL:    3600,
+  }
+  tempURL, err := objects.CreateTempURL(serviceClient, "{containerName}", "{objectName}", createTempURLOpts)
```
Comment (smashwilson) [inline src/docs/cloud-files/getting-started/samples/get_object_temp_url.rst]: The other language samples include setting the temp URL key on your account. Can we do that yet?

(They also use "example_container" and "someobject" as the container and object names, respectively.)

Comment (jrperritt) [inline src/docs/cloud-files/getting-started/samples/get_object_temp_url.rst]: Oh, sure. 

I must've looked at the `sh` sample, which has {containerName} and {objectName}.

**Comment by jrperritt** [comment]
There we go.

**Comment by smashwilson** [comment]
:+1:

**Pull request merged**